src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryDate.cs

```diff
@@ -466,9 +596,12 @@ public static FormulaValue DateTimeFunction(IRContext irContext, NumberValue[] a
             }
         }
 
-        private static FormulaValue Now(IRContext irContext, FormulaValue[] args)
+        private static async ValueTask<FormulaValue> Now(EvalVisitor runner, EvalVisitorContext context, IRContext irContext, FormulaValue[] args)
```

Comment:
>async [](http://example.com/codeflow?start=23&length=5)

Why did we make this function asynchronous?

Reply:
To return a ValueTask<T>